client/src/utils/getFormattedEthValue.ts

```diff
@@ -19,31 +21,50 @@ export default function getFormattedEthValue({
   shouldIgnoreGwei = false,
   shouldIgnoreWei = false,
   numberOfDecimalPlaces = 4,
+  showShortFormat = false,
```

Comment:
Please add unit test for this parameter.

Reply:
Done